packages/client/tests/functional/_utils/getTestSuiteInfo.ts

```diff
@@ -176,6 +185,15 @@ export function getTestSuiteSchema(suiteMeta: TestSuiteMeta, matrixOptions: Name
     schema = schema.replace(defaultGeneratorMatch[0], `${defaultGeneratorMatch[0]}\n${previewFeaturesStr}`)
   }
 
+  // for PlanetScale and Vitess, we need to add `relationMode = "prisma"` to the schema
+  if (matrixOptions.relationMode && providerMatch !== null) {
```

Comment:
Aside: There is a future where relationMode is optional for both.

Reply:
Yes. I was hesitant to implement `relationModesForFlavor` as an object of lists instead of a single `relationMode` per provider, perhaps I should have done that. This can easily be integrated in the future.